### PR TITLE
Added the "big5" workload based on Elastic's comparison benchmark

### PR DESCRIPTION
**Pull request by gkamat**
### Description
Added the "big5" workload based on Elastic's comparison benchmark.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: big5/README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Before using this repository, ensure you have the following installed:
+
+- [OpenSearch](https://opensearch.org) (v2.11 or later)
+- [OpenSearch Benchmark](https://opensearch.org/docs/latest/benchmark) (v1.2 or later)
+- An existing datastream with data generated using the open-source tool [Elastic Integration Corpus Generator Tool](https://github.com/elastic/elastic-integration-corpus-generator-tool). The data should follow the expected document structure mentioned below.
+
+
+### Data Document Structure
+
+The datastream must already exist and the data be generated using the open-source tool [Elastic Integration Corpus Generator Tool](https://github.com/elastic/elastic-integration-corpus-generator-tool). The expected structure of the documents is specified in the "Data Document Structure" section of this README.
```
Comment (IanHoang) [inline big5/README.md:19]: `The expected structure of the documents is specified in the "Data Document Structure" section of this README.` -> `Below is the expected structure of the documents.`

File: big5/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        "type": "filebeat",
+        "version": "8.8.0"
+    }
+}
+
+```
+
+
+
+### The "Big 5" Areas
```
Comment (IanHoang) [inline big5/README.md:68]: Nit: For each of the operations, a link to the corresponding OpenSearch documentation to allow users to learn more would be useful.

**Review by IanHoang — APPROVED** [review]
LGTM

File: big5/README.md (hunk 1 above)
Comment (gkamat) [reply big5/README.md:19]: Good point.  Will change this in the next iteration.

File: big5/README.md (hunk 2 above)
Comment (gkamat) [reply big5/README.md:68]: Will add this as well.

**Pull request merged**